src/03_modules.py

```diff
@@ -10,22 +10,25 @@
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
-
+print(len(sys.argv))
```

Comment:
@AnniqueN: Here where you should have a 'for' loop that iterates through 'sys.argv' to print the arguments that you're passing to 'python' on the command line:

`for arg in sys.argv:`
`     print(arg)`